packages/flutter/lib/src/material/input_decorator.dart

```diff
@@ -1598,12 +1688,23 @@ class InputDecorator extends StatefulWidget {
 
   /// Whether the input field has focus.
   ///
-  /// Determines the position of the label text and the color and weight
-  /// of the border.
+  /// Determines the position of the label text and the color and weight of the
```

Comment:
Isn't this also blended with the `hoverColor` if `isHover` is true as well? Maybe just mention that is contributes the `focusColor` to the eventual fill color?

Reply:
Yes, good idea. I added a `See also:` section pointing to the `hoverColor` and `isHovering`.